src/nuclio/common/components/edit-item/edit-item.component.js

```diff
@@ -445,6 +434,30 @@
             return ctrl.type === 'volume' ? ctrl.item.volume.name : ctrl.item.name;
         }
 
+        /**
+         * Gets corresponding tooltip description
+         * @returns {string}
+         */
+        function getTooltip() {
+            var tooltips = {
```

Comment:
add this variable to `ctrl.tooltips`. You don't need to define it on each function call